css/style.css

```diff
@@ -0,0 +1,134 @@
+body {
+  margin: 0 auto;
+  padding-left: 5px;
+}
+.firstfield {
+  display: flex;
+  flex-wrap: wrap;
+}
+.firstfield > svg {
+  padding: 1px;
+}
+.kiwi-tag {
+  fill:orange;
+}
+.lake {
+  fill:blue;
+}
+.kiwi-object {
+  width: 150px;
+}
+.kiwi-base64 {
+  width: 150px;
+}
+/*for 3*/
+.circle-anim {
+  animation-duration: 2s;
+  animation-name: circleanim;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+  @keyframes circleanim {
```

Comment:
Unneeded spaces.
Also, think of a better animation name.

Reply:
Edited.